Sources/Public/ImageProvider/AnimationImageProvider.swift

```diff
@@ -26,10 +27,18 @@ public protocol AnimationImageProvider {
 
   /// The image to display for the given `ImageAsset` defined in the `LottieAnimation` JSON file.
   func imageForAsset(asset: ImageAsset) -> CGImage?
+
+  /// Specifies how the layer's contents are positioned or scaled within its bounds for a given asset.
+  func contentsGravity(for asset: ImageAsset) -> CALayerContentsGravity?
 }
 
 extension AnimationImageProvider {
   public var cacheEligible: Bool {
     true
   }
+
+  /// The default value is `.resize`, similar to that of `CALayer`.
+  public func contentsGravity(for asset: ImageAsset) -> CALayerContentsGravity? {
+    nil
+  }
```

Comment:
Maybe it would be better for this to be non-optional, with the default defined like this:

```suggestion
  public func contentsGravity(for asset: ImageAsset) -> CALayerContentsGravity {
    .resize
  }
```